.github/workflows/build.yml

```diff
@@ -138,14 +144,15 @@ jobs:
             eval "$bk"
           fi
           eval "$shell"
+
       - name: Push
         env:
           DOCKER_HUB_USERNAME: ${{ secrets.DOCKER_HUB_USERNAME }}
           DOCKER_HUB_PASSWORD: ${{ secrets.DOCKER_HUB_PASSWORD }}
         run: |
           export DOCKER_CONFIG="$PWD/.docker"
           mkdir "$DOCKER_CONFIG"
-          trap 'rm -rf "$DOCKER_CONFIG"' EXIT
+          trap 'find "$DOCKER_CONFIG" -type f -exec shred -fuvz "{}" + || :; rm -rf "$DOCKER_CONFIG"' EXIT
```

Comment:
Is the `rm -rf "$DOCKER_CONFIG"` necessary since `shred -u` is used?

Reply:
It is not strictly necessary, no -- this is mostly me trying to be extra cautious/careful about making sure these bits do get removed from disk in the safest way we can (at least for the "on-disk" bits that are our fault, not anything GitHub might've put on-disk that we don't know about).